django/http/request.py

```diff
@@ -230,6 +288,17 @@ def body(self):
         if not hasattr(self, '_body'):
             if self._read_started:
                 raise RawPostDataException("You cannot access body after reading from request's data stream")
+
+            # Limit the maximum request data size that will be handled in-memory.
+            try:
+                content_length = int(self.META.get('HTTP_CONTENT_LENGTH', self.META.get('CONTENT_LENGTH', 0)))
```

Comment:
''' content_length = int(self.META.get('HTTP_CONTENT_LENGTH', self.META.get('CONTENT_LENGTH', 0)))'''

I guess this is correct, but it is not obvious to me when HTTP_CONTENT_LENGTH (I did not find where it could be set) is set and when it falls back to CONTENT_LENGTH. And what is the difference. Perhaps this deserves a comment.